api/v1beta1/azurecluster_webhook.go

```diff
@@ -45,21 +41,16 @@ var _ webhook.Defaulter = &AzureCluster{}
 
 // Default implements webhook.Defaulter so a webhook will be registered for the type.
 func (c *AzureCluster) Default() {
-	clusterlog.Info("default", "name", c.Name)
-
 	c.setDefaults()
 }
 
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type.
 func (c *AzureCluster) ValidateCreate() error {
-	clusterlog.Info("validate create", "name", c.Name)
```

Comment:
are we loosing any valuable information by removing these logs? would it make sense to downgrade them to v() > 1 instead?

Reply:
I didn't see this kind of logging in CAPI, so I ripped them out. I'm happy to add them back if they add value.